pkg/tlsf/.gitattributes

```diff
@@ -1,2 +0,0 @@
-# Ignore whitespace errors inside these patch files by regarding them as binary.
-patch.txt binary
```

Comment:
> A seperate repository does not help me if I want to re-review changes to the package (i.e. the patch files) in the RIOT-repository. 

Why not?

For me a diff of a diff is close enough to binary representation. Let's discuss on Tuesday.